packages/ember-data/lib/initializers/store.js

```diff
@@ -12,13 +12,20 @@ import Store from "ember-data/system/store";
   @param {Object} [application] an application namespace
 */
 export default function initializeStore(registry, application) {
+  // initializerStore be called by an Ember.initializer
+  // when used with ember 1.11.0 or below.
+  var params = getRegistryAndContainer(registry, application);
+  application = params.application;
```

Comment:
it's a bit confusing to override passed in arguments, perhaps name variables differently?